### ISSUE TITLE
Slow Rendering Of Text in Powershell in Windows Terminal

### ISSUE THREAD

**Issue by Rasesh2005** [microsoft/terminal#6399]
# Environment

```none
Windows build number: [run `[Environment]::OSVersion` for powershell, or `ver` for cmd]
Windows Terminal version (if applicable):

Any other software?
```

# Steps to reproduce

# Expected behavior

# Actual behavior

When I complete typing my whole command and look at the screen it has started typing The first word.Sowly typing letter by letter.

I don't have any such problem on opening powershell from win + x But I love the design Of terminal,Therefore came here to report the issue.

By the way, Command Prompt(cmd.exe) is working absolutely fine but i am more comfortable with interface of windows powershell.

Please Fix this bug as soon as possible

I Dont Have any Recording Tool To Attach The Recording.So,sorry couldn't give proof

Summing up, The problem is  only Windows Powershell is lagging a lot while typing in windows terminal.But runs Perfectly Fine While opened as Standalone(form win + x )
